_packages/native-preview/README.md

```diff
@@ -0,0 +1,18 @@
+# TypeScript (Native Preview)
+
+[Not sure what this is? Read the announcement post!](https://devblogs.microsoft.com/typescript/typescript-native-port/)
```

Comment:
At some point I think we should update the text here. Maybe that point is now?

Is this what people will see on npm? How do people feel about naming this `npm-native-preview`?

Reply:
Yes, this is what people will see on npm, so worth making look good.

Everything in this dir is intended to be published to npm, so it seems redundant to call it that.

Reply:
>Everything in this dir is intended to be published to npm, so it seems redundant to call it that.

Right, but I don't know if that's obvious for someone exploring our codebase for the first time.

Reply:
Cynically, if they're exploring this repo for the first time, the last place I hope they look is the build infrastructure 😄